{"name": "test_simple_assume9", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n2 Failed Assumptions:\n\ntest_xdist.py:46: AssumptionFailure\n>>\tpytest.assume(1==0)\nAssertionError: assert False\n\ntest_xdist.py:48: AssumptionFailure\n>>\tpytest.assume(False)\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               2 Failed Assumptions:\nE               \nE               test_xdist.py:46: AssumptionFailure\nE               >>\tpytest.assume(1==0)\nE               AssertionError: assert False\nE               \nE               test_xdist.py:48: AssumptionFailure\nE               >>\tpytest.assume(False)\nE               AssertionError: assert False\n\nD:\\Miniconda\\Lib\\site-packages\\six.py:718: FailedAssumption"}, "start": 1701430294661, "stop": 1701430294704, "uuid": "aa7ff090-7929-401f-9886-72b3eb8efe2e", "historyId": "7fe836d134ded8e2a134f3a975b33b1a", "testCaseId": "7fe836d134ded8e2a134f3a975b33b1a", "fullName": "test_xdist#test_simple_assume9", "labels": [{"name": "suite", "value": "test_xdist"}, {"name": "host", "value": "QuMoSi"}, {"name": "thread", "value": "14004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_xdist"}]}